SoftLayer/managers/file.py

```diff
@@ -22,9 +22,10 @@ def list_file_volume_limit(self):
         """
         return self.get_volume_count_limits()
 
-    def list_file_volumes(self, datacenter=None, username=None, storage_type=None, **kwargs):
+    def list_file_volumes(self, datacenter=None, username=None, order=None, storage_type=None, **kwargs):
```

Comment:
the new parameter order should go after storage_type, not before. This way anyone passing in arguments without the names wont encounter an error.

Reply:
Done.